core/controllers/topic_viewer.py

```diff
@@ -124,9 +124,10 @@ def get(self, topic_name):
 
         train_tab_should_be_displayed = False
         if all_skill_ids:
-            questions = question_services.get_questions_by_skill_ids(
-                5, all_skill_ids, False)
-            if len(questions) == 5:
+            question_count = (
+                question_services.get_total_question_count_for_skill_ids(
```

Comment:
While your changes are correct, I think this underlying function may return the wrong count. (Correct me if I am wrong though @aks681 !) One question can be mapped to multiple skills and looking at the underlying function that returns the count : https://github.com/oppia/oppia/blob/a98aa9e0385bc9eca535d4eb37759276620c1492/core/storage/question/gae_models.py#L266 it doesnt seem to be removing duplicate question ids. Can we make that change here as well?

Reply:
Actually, thinking about it, I think we can merge this as-is? It's true that it overcounts, but I think the hotfix should be as small a change as possible, and I think it'll be fine in practice. (Also in the future we should just change this to a boolean variable associated with the topic, so that there aren't so many queries on a GET call.)

How does this sound @vinitamurthi?

Also @nithusha21 please could you file an issue for speeding up the topic viewer GET call in general, and mention it here? Thanks!

Reply:
Ok I'll do that. I don't think overcounting matters for the hotfix because as Sean mentioned we don't have data for questions right? I'll file an issue for the speedup of the topic viewer call.

Reply:
https://github.com/oppia/oppia/issues/10242

Reply:
Sounds good! Approving this to unblock